api/v1alpha1/instancegroup_types.go

```diff
@@ -187,26 +187,28 @@ type EKSSpec struct {
-	BootstrapArguments          string                    `json:"bootstrapArguments,omitempty"`
-	SpotPrice                   string                    `json:"spotPrice,omitempty"`
-	Tags                        []map[string]string       `json:"tags,omitempty"`
-	Labels                      map[string]string         `json:"labels,omitempty"`
-	Taints                      []corev1.Taint            `json:"taints,omitempty"`
-	UserData                    []UserDataStage           `json:"userData,omitempty"`
-	ExistingRoleName            string                    `json:"roleName,omitempty"`
-	ExistingInstanceProfileName string                    `json:"instanceProfileName,omitempty"`
-	ManagedPolicies             []string                  `json:"managedPolicies,omitempty"`
-	MetricsCollection           []string                  `json:"metricsCollection,omitempty"`
-	LifecycleHooks              []LifecycleHookSpec       `json:"lifecycleHooks,omitempty"`
-	MixedInstancesPolicy        *MixedInstancesPolicySpec `json:"mixedInstancesPolicy,omitempty"`
+	EksClusterName              string                       `json:"clusterName,omitempty"`
+	KeyPairName                 string                       `json:"keyPairName,omitempty"`
+	Image                       string                       `json:"image,omitempty"`
+	InstanceType                string                       `json:"instanceType,omitempty"`
+	NodeSecurityGroups          []string                     `json:"securityGroups,omitempty"`
+	Volumes                     []NodeVolume                 `json:"volumes,omitempty"`
+	Subnets                     []string                     `json:"subnets,omitempty"`
+	SuspendedProcesses          []string                     `json:"suspendProcesses,omitempty"`
+	BootstrapArguments          string                       `json:"bootstrapArguments,omitempty"`
+	SpotPrice                   string                       `json:"spotPrice,omitempty"`
+	Tags                        []map[string]string          `json:"tags,omitempty"`
+	Labels                      map[string]string            `json:"labels,omitempty"`
+	Taints                      []corev1.Taint               `json:"taints,omitempty"`
+	UserData                    []UserDataStage              `json:"userData,omitempty"`
+	ExistingRoleName            string                       `json:"roleName,omitempty"`
+	ExistingInstanceProfileName string                       `json:"instanceProfileName,omitempty"`
+	ManagedPolicies             []string                     `json:"managedPolicies,omitempty"`
+	MetricsCollection           []string                     `json:"metricsCollection,omitempty"`
+	LifecycleHooks              []LifecycleHookSpec          `json:"lifecycleHooks,omitempty"`
+	MixedInstancesPolicy        *MixedInstancesPolicySpec    `json:"mixedInstancesPolicy,omitempty"`
+	LicenseSpecifications       []string                     `json:"licenseSpecifications,omitempty"`
```

Comment:
Are both theses features `LicenseSpecifications` and `Placement` support by both LaunchTemplate and LaunchConfig?

Reply:
Unfortunately no, they're specific to LaunchTemplates.